.github/workflows/unit.yaml

```diff
@@ -18,7 +18,7 @@ jobs:
         ref: ${{ inputs.ref }}
 
     - name: Set up Go
-      uses: actions/setup-go@v2
+      uses: actions/setup-go@v3
```

Comment:
This to fix a warning that we are using deprecated action, so updated this to v3.